### PR TITLE
[icon] a11y

### PR DESCRIPTION
**Pull request by phytonmk**
Made aria-label required for interactive icons

## Description

If `interactive` prop is provided for icon but `aria-label` or `aria-labelledby` are not, a warning is logged in non-production env.

Non-interactive icons are aria-hidden now.

## How has this been tested?

I have not added any new tests but updated lot of them (by adding aria-label). Suppose if now axe errors are appeared and no new warnings during jest run changes may be considered as working.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by freeyoungstrong — APPROVED** [review]
LGTM

File: semcore/pagination/src/Pagination.jsx [hunk 1]
```diff
@@ -306,9 +306,11 @@ class PageInput extends Component {
           ) : (
             <>
               <Pagination.PageInput.Value id={`pagination-input-${uid}`} />
-              <Pagination.PageInput.Addon interactive>
-                <Return />
-              </Pagination.PageInput.Addon>
+              <Pagination.PageInput.Addon
+                tag={Return}
+                interactive
+                aria-label="Confirm page number"
```
[the remark this answers is not in the document]
Comment (phytonmk) [reply semcore/pagination/src/Pagination.jsx:312]: I also was surprised but it is a actual Icon behaviour in pagination component. Maybe you have some better ideas for that label?

[the remark this answers is not in the document]
Comment (phytonmk) [reply semcore/pagination/src/Pagination.jsx:312]: As far as I see from [documentation examples](https://developer.semrush.com/intergalactic/components/pagination/pagination-code/) return icon behaviour is same to  behaviour of Enter key press (value confirmation). Am I wrong?

[the remark this answers is not in the document]
Comment (ekrasnopolskaia) [reply semcore/pagination/src/Pagination.jsx:312]: Hm, yeah, you right, sorry

**Review by ekrasnopolskaia — APPROVED** [review]

**Pull request merged**